test/language.c

```diff
@@ -99,5 +99,15 @@ int main(void)
     TEST_LANG_GUESS("foo(en-US)(sdh).srt", "en-US", 3, true);
     TEST_LANG_GUESS("foo().srt", "", -1, false);
 
+    TEST_LANG_GUESS("foo.bar.srt", "", 3, false);
```

Comment:
I think we should reset pos to -1 if there is no match.

Reply:
Yeah that makes more sense.